Makefile

```diff
@@ -126,8 +126,8 @@ comma:= ,
 semicolon:= ;
 
 define create_mock
-  mkdir -p $(1)/gomocks && rm -rf $(1)/gomocks/*
-  $(MOCKGEN) -destination $(1)/gomocks/mocks.go -self_package mocks -package mocks $(PROJECT_ROOT)/$(1) $(subst $(semicolon),$(comma),$(2))
+  rm -rf $(1)/gen_mocks.go
+  $(MOCKGEN) -destination $(1)/gen_mocks.go -self_package $(PROJECT_ROOT)/$(1) -package $(2) $(PROJECT_ROOT)/$(1) $(subst $(semicolon),$(comma),$(3))
```

Comment:
Mocks do not belong in the same package as the implementation.

Reply:
Done!